cfy_manager/components/postgresql_server/postgresql_server.py

```diff
@@ -482,10 +485,10 @@ def _get_patroni_dcs_conf(self, local_only=True):
         ).aggr_stdout)
 
     def _get_etcd_id(self, ip):
-        return 'etcd' + ip.replace('.', '_')
+        return 'etcd' + ip.replace('.', '_').replace(':', '_')
 
     def _get_patroni_id(self, address):
-        return 'pg' + address.replace('.', '_')
+        return 'pg' + address.replace('.', '_').replace(':', '_')
```

Comment:
Do you think it's worth us making a function for this somewhere? We seem to have it in several places.

Reply:
I was thinking the same, but we other places are (mostly?) Jinja2 templates, so without overcomplicating I deemed this to be just a simpler way of dealing with id creation.  I'm not completely reluctant to revising my point however :)